janusgraph-cassandra/src/test/java/org/janusgraph/diskstorage/cassandra/AbstractCassandraStoreTest.java

```diff
@@ -43,6 +44,7 @@
     private static final String DEFAULT_COMPRESSOR_PACKAGE = "org.apache.cassandra.io.compress";
 
     public abstract ModifiableConfiguration getBaseStorageConfiguration();
+    public abstract ModifiableConfiguration getBaseStorageConfiguration(String keyspace);
```

Comment:
I am very confused because this came in from #412 and both the build there and master now are not failing